nemo_skills/pipeline/run_cmd.py

```diff
@@ -13,21 +13,26 @@
 # limitations under the License.
 
 import logging
-from typing import List
+from typing import List, Optional
```

Comment:
Would be great to update documentation over here https://github.com/NVIDIA/NeMo-Skills/blob/main/docs/pipelines/run-cmd.md to explain the new server arguments and how the command interacts with them